packages/extension/src/store/synchnorousCookieStore.ts

```diff
@@ -40,6 +46,51 @@ class SynchnorousCookieStore {
     };
   } = {};
 
+  /**
+   * The cookie data of the tabs.
+   */
+  tabMode: 'single' | 'unlimited' = 'single';
+
+  /**
+   * CookieDatabase to run analytics match on.
+   */
+  cookieDB: CookieDatabase | null = null;
+
+  /**
+   * The cookie data of the tabs.
+   */
+  tabToRead = '';
+
+  requestIdToCDPURLMapping: {
```

Comment:
Adding JSdoc will help, and for the below variables too.

Reply:
Fixed in latest commits.